hugegraph-core/src/main/java/com/baidu/hugegraph/backend/tx/GraphIndexTransaction.java

```diff
@@ -249,8 +251,8 @@ protected void updateIndex(Id ilId, HugeElement element, boolean removed) {
                     }
                 } else {
                     for (int i = 0, n = propValues.size(); i < n; i++) {
-                        List<Object> prefixValues =
-                                propValues.subList(0, i + 1);
+                        List<Object> prefixValues = propValues
+                                .subList(0, i + 1);
```

Comment:
prefer to wrap line after "=" and align propValues.subList(0, i + 1) with prefixValues